### ISSUE TITLE
PHP8 release

### ISSUE THREAD

**Issue by arjanwestdorp** [irazasyed/laravel-gamp#34]
Thanks for you great package! 

Would it be possible to release a version with PHP8 support? I see that the master branch supports PHP8, but there is not version to reference it yet.

**Comment by jLynx** [comment]
+1 for this please!

**Comment by jLynx** [comment]
Update: I was able to get it to work with:
```
"repositories": [
	{ "type": "vcs", "url": "https://github.com/irazasyed/laravel-gamp" }
],
"require": {
	"irazasyed/laravel-gamp": "master",
},
```

**Comment by irazasyed** [comment]
Released `v1.8` 

Please try it out and create a new ticket if there are any issues.
